xrmocap/model/loss/prior_loss.py

```diff
@@ -179,6 +193,47 @@ def forward(self,
                 torch.tensor([1., -1., -1, -1.], device=body_pose.device)) -
                                 1)**2
 
+        if self.lock_foot:
```

Comment:
What about merge the three `if`s into a for loop?
```python
parts_joint_prior_losses = []
pred_poses = []
limits = []
weights = []
if self.use_full_body:
    pred_poses.append(aa_to_sja(body_pose_reshape, self.R_t, self.R_t_inv))
    limits.append(self.sja_limits)
    weights.append(1.0)
if self.lock_foot:
    pred_poses.append(body_pose_sja[:, [6, 7, 9, 10]])
    limits.append(self.sja_lock_foot)
    weights.append(self.lock_foot_loss_weight)
...
for idx, weight in enumerate(weights):
    lower_limits = limits[idx][:, :, 0]
    upper_limits = limits[idx][:, :, 1]
    ...
    parts_joint_prior_losses.append(weight * loss)


```

Reply:
Refactored in [cab8731](https://github.com/openxrlab/xrmocap/pull/56/commits/cab87319d90d6f7f6e5f611602f0b4c4d7d6f938).